core/imagedownloader.cpp

```diff
@@ -154,19 +154,9 @@ Thumbnailer::Thumbnail Thumbnailer::getHashedImage(const QString &filename, bool
 	return thumbnail;
 }
 
-static QImage renderIcon(const char *id, int size)
-{
-	QImage res(size, size, QImage::Format_RGB32);
-	res.fill(Qt::white);
-	QSvgRenderer svg{QString(id)};
-	QPainter painter(&res);
-	svg.render(&painter);
-	return res;
-}
-
-// As renderIcon, but render to a fixed width and scale height accordingly
+// As renderSVGIcon(), but render to a fixed width and scale height accordingly
 // and have a transparent background.
-static QImage renderIconWidth(const char *id, int size)
+static QImage renderSVGIconWidth(const char *id, int size)
```

Comment:
I wonder if it wouldn't be more consistent to move this function into qthelper as well - those two seem rather closely related, having one here, one there seems... odd

Reply:
done. That said, qthelper.cpp really needs to be split up.